src/pages/Defi/hooks/useFoxyBalances.tsx

```diff
@@ -1,65 +0,0 @@
-import { ethChainId, foxAssetId, fromAccountId, toAccountId } from '@shapeshiftoss/caip'
-import { useMemo } from 'react'
-import { useWallet } from 'hooks/useWallet/useWallet'
-import { useWalletSupportsChain } from 'hooks/useWalletSupportsChain/useWalletSupportsChain'
-import { isValidAccountNumber } from 'lib/utils'
-import { useGetFoxyAprQuery, useGetFoxyBalancesQuery } from 'state/apis/foxy/foxyApi'
-import {
-  selectPortfolioAccountIdsByAssetId,
-  selectPortfolioAccountsGroupedByNumberByChainId,
-} from 'state/slices/portfolioSlice/selectors'
-import { useAppSelector } from 'state/store'
-
-export function useFoxyBalances({ accountNumber }: { accountNumber?: number } = {}) {
-  const {
-    state: { wallet },
-  } = useWallet()
-
-  const accountsByNumber = useAppSelector(state =>
-    selectPortfolioAccountsGroupedByNumberByChainId(state, { chainId: ethChainId }),
-  )
-
-  const userAddress = useMemo(() => {
-    if (!isValidAccountNumber(accountNumber)) return null
-    const accountId = accountsByNumber[accountNumber]?.[0] // Only one address per account for EVM chains i.e no multiple accountTypes
-    return accountId ? fromAccountId(accountId).account : null
-  }, [accountNumber, accountsByNumber])
-
-  const { data: foxyAprData } = useGetFoxyAprQuery()
-
-  const supportsEthereumChain = useWalletSupportsChain({ chainId: ethChainId, wallet })
```

Comment:
Not needed anymore - we will fetch metadata no matter the wallet connected, and won't fetch userData if no Ethereum AccountIds are present